plugins/tracing/tracing.go

```diff
@@ -38,12 +39,16 @@ func init() {
 // ExternalConfigName is the external config name of tracing.
 const ExternalConfigName = "tracing"
 
-// config is tracing config.
+// RequestHook allows you to custom information for span.
+type RequestHook func(ctx context.Context, req *http.Request, span opentracing.Span)
```

Comment:
The comments have little information. 

And a default hook for `X-Request-ID` is desired.